samples/hello++/main.cpp

```diff
@@ -1,27 +1,21 @@
 #include <hal/debug.h>
 #include <hal/video.h>
+#include <string>
+#include <vector>
```

Comment:
I was unsure in which order I should put the includes. Settled for: nxdk-specific over cpp-specific and alphabetical.